sdk/search/Azure.Search.Documents/tests/Samples/Sample06_EncryptedIndex.cs

```diff
@@ -120,25 +126,35 @@ public async Task CreateDoubleEncryptedIndex()
                     Environment.GetEnvironmentVariable("SEARCH_API_KEY"));
 
                 SearchIndexerClient indexerClient = new SearchIndexerClient(endpoint, credential);
-                /*@@*/ indexerClient = resources.GetIndexerClient();
+#if !SNIPPET
+                indexerClient = resources.GetIndexerClient();
+#endif
```

Comment:
I think we can reorder lines and merge ifs

Reply:
There's a few places, yeah (and I fixed a few more), but this one, for example, has ordering dependencies e.g. lines that need this below.